sh/tests/my-ls_test.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+# Unofficial Bash Strict Mode
+set -euo pipefail
+IFS='
+'
+
+FILENAME="student/my-ls.sh"
+script_dirS=$(cd -P "$(dirname "$BASH_SOURCE")" &>/dev/null && pwd)
+
+challenge() {
+    unalias my-ls
```

Comment:
the test gets stuck here even using your solution.
"./my-ls_test.sh: line 12: unalias: my-ls: not found
"
I think is because the alias is never saved in the bashrc file with that solution

Reply:
yes fixed cause a new env will created to exec this script, so the alias is not exists

Reply:
still the test doesn't run:
![image](https://user-images.githubusercontent.com/53977751/209707204-bd874242-d01e-4aef-89a3-2e3e70339a67.png)
and i'm using your solution as the student solution so they are the same